lib/rgb_matrix/engine.ex

```diff
@@ -9,11 +9,12 @@ defmodule RGBMatrix.Engine do
   alias Layout.LED
   alias RGBMatrix.Animation
 
-  @type frame :: %{LED.t() => RGBMatrix.any_color_model()}
+  @type frame :: %{led_id => RGBMatrix.any_color_model()}
+  @type led_id :: atom
```

Comment:
Well, we can rework the type here, or we can rework all the animations and other functions which deal with frames, because from what I can see, everything actually seems to use a `%{atom => Color}` format...